### PR TITLE
feat:add x6 sites base on gh-pages

### PR DESCRIPTION
**Pull request by NewByVector**
### Description
add x6 sites base on gh-pages

### Motivation and Context
Create official website

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Enhancement (changes that improvement of current feature or performance)
- [ ] Refactoring (changes that neither fixes a bug nor adds a feature)
- [ ] Test Case (changes that add missing tests or correct existing tests)
- [ ] Code style optimization (changes that do not affect the meaning of the code)
- [x] Docs (changes that only update documentation)
- [ ] Chore (changes that don't modify src or test files)

### Self Check before Merge
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/antvis/x6/blob/master/CONTRIBUTING.md) document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by NewByVector** [comment]
feat:add x6 sites base on gh-pages

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,5 @@ coverage
 lib
 es
 dist
+public
```
Comment (bubkoo) [inline .gitignore:11]: 在 x6-sites 本地仓库添加一个 .gitignore 文件吧

Comment (bubkoo) [reply .gitignore:11]: 顺便文件夹也改成单数形式 `x6-sites` => `x6-site`

**Comment by afc163** [comment]
image

邮箱没绑定对。

**Pull request merged**